Common/UnitDefinitions/Force.json

```diff
@@ -156,11 +156,11 @@
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "kipf" ]
+          "Abbreviations": [ "kipf", "kip", "k" ]
         },
         {
           "Culture": "ru-RU",
-          "Abbreviations": [ "кипф" ]
+          "Abbreviations": [ "кипф", "койка", "К" ]
```

Comment:
I'll just trust you on this one

Reply:
And i trusted google translate on this one :x

PS: Don't forget to release a new nuget? 😄

Reply:
Forgot, thanks. Done!